test/indexer/test_hca_indexer.py

```diff
@@ -870,6 +870,25 @@ def test_cell_line_sample(self):
                 self.assertEqual(one(contents['cell_suspensions'])['organ'], ['blood (child_cell_line)'])
                 self.assertEqual(one(contents['cell_suspensions'])['organ_part'], [self.translated_str_null])
 
+    def test_multiple_samples(self):
+        """
+        Index a bundle with a specimen_from_organism and a cell_line input into
+        a cell_suspension resulting in two samples of different entity_type
+        """
+        self._index_canned_bundle(('1b6d8348-d6e9-406a-aa6a-7ee886e52bf9', '2019-10-03T105524.911627Z'))
+        hits = self._get_all_hits()
+        for hit in hits:
+            contents = hit['_source']['contents']
+            entity_type, aggregate = config.parse_es_index_name(hit['_index'])
+            cell_suspension = one(contents['cell_suspensions'])
+            self.assertEqual(cell_suspension['organ'], ['embryo', 'immune system'])
+            self.assertEqual(cell_suspension['organ_part'], ['skin epidermis', self.translated_str_null])
+            if aggregate and entity_type != 'samples':
+                self.assertEqual(one(contents['samples'])['entity_type'], [
```

Comment:
I am wondering if this could benefit from `self.subTest` I'll leave it to your discretion.

Reply:
The main purpose of the test is for the `cell_suspension['organ_part']` value, however since I am including a verification of the `contents['samples'][]['entity_type']` value I agree it should be checked in more cases than the current `if` block. I'll add the missing `else`.

Reply:
Yes, I don't think a subtest is needed here.